core/packages/test/scripts/helpers/mutateSpec.js

```diff
@@ -20,6 +24,7 @@ function run() {
     let initialSync = JSON.parse(fs.readFileSync(process.argv[3]));
 
     data['genesis']['runtime']['ethereumBeaconClient']['initialSync'] = initialSync;
+    data['genesis']['runtime']['ethereumBeaconClient']['initialSync']['import_time'] = getTimestampInSeconds()
```

Comment:
That's a good point. Although if we take longer than 27 hours to deploy and future date the import time, it opens up the risk of a long range attack because the initial sync checkpoint header is older than 27 hours. But maybe that's OK, even if it blocks the bridge on deployment, and then we provide another checkpoint through governance. Thanks for picking this up, I'll remember to take this into account with the new bridge hub architecture. 🤞